cmd/zdb/zdb.c

```diff
@@ -6205,8 +6205,8 @@ zdb_check_for_obsolete_leaks(vdev_t *vd, zdb_cb_t *zcb)
 		    inner_offset < DVA_GET_ASIZE(&vimep->vimep_dst);
 		    inner_offset += 1 << vd->vdev_ashift) {
```

Comment:
Why not this one too?

Reply:
That was an oversight. I will add it and repush.

Reply:
Fixed.